src/AccessibilityInsights.SharedUx/SharedUx.csproj

```diff
@@ -29,6 +29,8 @@
 
   <ItemGroup>
     <None Remove="Resources\Icons\BrandIcon.ico" />
+    <None Remove="Resources\Sound\start_event_recording.wav" />
```

Comment:
Should two references to this new resource be added?

Reply:
Yes, the resource should have both Remove and Include. I'd suggest that we follow the pattern that the IDE uses to add resources, and create a new ItemGroup for each resource. That keeps everything about each file in one place, instead of scattering it around. It would look something like this:

```
<ItemGroup>
  <None Remove="Resources\Sound\start_event_recording.wav" />
  <EmbeddedResource Include="Resources\Sound\start_event_recording.wav" />
</ItemGroup>

<ItemGroup>
  <None Remove="Resources\Sound\stop_event_recording.wav" />
  <EmbeddedResource Include="Resources\Sound\stop_event_recording.wav" />
</ItemGroup>
```